client-src/elements/form-field-enums.js

```diff
@@ -42,6 +64,9 @@ export const FEATURE_TYPES = {
     'This type of feature entry can be referenced from a PSA immediately.'],
   FEATURE_TYPE_DEPRECATION_ID: [3, 'Feature deprecation',
     'Deprecate and remove an old feature.'],
+  FEATURE_TYPE_ENTERPRISE_ID: [4, 'New Feature or removal affecting enterprises',
+    'For features or changes that need to be communicated to enterprises or schools',
+    'enterprisefeature'],
```

Comment:
Where is this used?

Reply:
Good catch, it is a remnant of a test I made.